src/commands/basemaps-github/create-pr.ts

```diff
@@ -13,7 +13,15 @@ import { Category, MakeCogGithub } from './make.cog.github.js';
 const validTargetBuckets: Set<string> = new Set(['linz-basemaps', 'linz-basemaps-staging']);
 const validSourceBuckets: Set<string> = new Set(['nz-imagery', 'linz-imagery']);
 
-async function parseTargetInfo(
+async function validateBucket(bucket: string, validBuckets: Set<string>): Promise<void> {
```

Comment:
nit: this is a assertion function as it throws on error

`assertValidBucket(..)`

Reply:
it also should not be async

Reply:
Thanks, just updated.